config/values-schema.yml

```diff
@@ -17,3 +17,57 @@ dangerous_enable_pprof: false
 tls_cipher_suites: ""
 #@schema/desc "API port"
 api_port: 10350
+#@schema/desc "The coreDNSIP will be injected into /etc/resolv.conf of kapp-controller pod"
+coreDNSIP: ""
+#@schema/desc "HostNetwork of kapp-controller deployment."
+hostNetwork: false
+#@schema/desc "PriorityClassName of kapp-controller deployment."
+priorityClassName: ""
+#@schema/desc "Concurrency of kapp-controller deployment"
+concurrency: 0
```

Comment:
I used 0 to not make any change to our current release artefact. But this would also result in a default value of 0 for concurrency, which would be weird.